modules.d/35network-manager/module-setup.sh

```diff
@@ -51,9 +51,12 @@ install() {
         inst_simple "$moddir"/nm-initrd.service "$systemdsystemunitdir"/nm-initrd.service
         inst_simple "$moddir"/nm-wait-online-initrd.service "$systemdsystemunitdir"/nm-wait-online-initrd.service
 
-        # Adding default link
-        inst_multiple -o "${systemdutildir}/network/99-default.link"
-        [[ $hostonly ]] && inst_multiple -H -o "${systemdsystemconfdir}/network/*.link"
+        # Add default link if there is no persistent network device naming
+        if [ ! -e /etc/udev/rules.d/70-persistent-net.rules ]; then
+
+            inst_multiple -o "${systemdutildir}/network/99-default.link"
+            [[ $hostonly ]] && inst_multiple -H -o "${systemdnetworkconfdir}/*.link"
```

Comment:
Sorry to bother you again with this, it's a pain patching old versions...

`${systemdnetworkconfdir}` is also wrong defined in this version:

https://github.com/openSUSE/dracut/blob/dcb9bdbf70650051957e2b93863ab14aab87c47e/dracut.sh#L1728

Upstream patch: https://github.com/dracutdevs/dracut/pull/1868/commits/ae4c8744

In summary, you can use:

- Main conf dir: `${systemdutildir}/network`
- Local conf dir: `${systemdutilconfdir}/network`

Reply:
Thanks Antonio, better I fix everything now than getting bug reports later.
I've also updated it in the network-legacy module.